### ISSUE TITLE
`reify` doesn't work when model has an enum attribute

### ISSUE THREAD

**Issue by hicklin-james** [liveeditor/draftsman#34]
I have a model defined like this:

```
class Question < ActiveRecord::Base
    ...
    enum question_response_type: { radio: 0, text: 1 }
    ...
end
```

When I try to call the `reify` method on a question draft, ActiveRecord throws an error:

`ArgumentError: You tried to define an enum named "question_response_type" on the model "Question", but this will generate a instance method "question_response_type=", which is already defined by another enum.`

Is there any way around this?

**Comment by dedman** [comment]
For anyone else googling this issue, this is now fixed in the draftsman 0.3.7 release. 

https://github.com/liveeditor/draftsman/releases/tag/v0.3.7

Thank you @bdunham.

**Comment by chrisdpeters** [comment]
:+1: Thanks for reporting, @dedman.

I would really like to get some tests in there for this. I will open a new issue to make sure that happens.
